codes/12865_py3_sjskgks.py

```diff
@@ -0,0 +1,18 @@
+import sys
+input = sys.stdin.readline
+
+numOfThings, limit = map(int, input().split())
+knapsack = [[0,0] for _ in range(limit+1)]
+past_pos, cur_pos = 0, 1
+for _ in range(numOfThings):

```

Comment:
past_pos, cur_pos를 뺴고 for문에서 이를 계산해보시겠어요?
```suggestion
for i in range(numOfThings):
    #past_pos, cur_pos = i % 2, (i + 1) % 2
```

Reply:
그러면 아래도 다 수정해야겠죠?